src/ts/config/config.ts

```diff
@@ -710,8 +709,8 @@ export namespace LayoutConfig {
                 show,
                 popout: header?.popout ?? labels?.popout ??
                     (settings?.showPopoutIcon === false ? false : ResolvedLayoutConfig.Header.defaults.popout),
-                dock: header?.popin ?? labels?.popin ?? ResolvedLayoutConfig.Header.defaults.dock,
-                maximise: header?.maximise ?? labels?.maximise ??
+                    dock: header?.popin ?? labels?.popin ?? ResolvedLayoutConfig.Header.defaults.dock,
```

Comment:
dock is leftover? 
is the indentation intended?

Reply:
ah yeah, this confused me, but this use of 'dock' actually means 'popin'.
The indentation is not intended.  Not sure why this happened, maybe because I overzealously removed it and later added it back.  I'll undo this change.